snippets/node-icsp-no-drain.adoc

```diff
@@ -0,0 +1,20 @@
+// Text snippet included in the following modules:
+//
+// * modules/about-crio.adoc
+// * modules/nodes-containers-using.adoc
+
+:_content-type: SNIPPET
+
+The following modifications do not trigger a node reboot:
+
+* When the MCO detects any of the following changes, it applies the update without draining or rebooting the node:
+
+** Changes to the SSH key in the `spec.config.passwd.users.sshAuthorizedKeys` parameter of a machine config.
+** Changes to the global pull secret or pull secret in the `openshift-config` namespace.
+** Automatic rotation of the `/etc/kubernetes/kubelet-ca.crt` certificate authority (CA) by the Kubernetes API Server Operator.
+
+* When the MCO detects changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageDigestMirrorSet` or `ImageTagMirrorSet` object, it drains the corresponding nodes, applies the changes, and uncordons the nodes.The node drain does not happen for the following changes:
```

Comment:
@mburke5678 `ImageDigestMirrorSet ` and `ImageTagMirrorSet` are new resource to us. we need to verify the behavior mentioned in the doc

Reply:
I have verified the behavior today:

- `ImageDigestMirrorSet` applies the mirrors with `pull-from-mirror = "digest-only"` and the drain operation is skipped.

- `ImageTagMirrorSet` applies the mirrors with `pull-from-mirror = "tag-only"` and it performs a drain operation as expected.